### ISSUE TITLE
SA1121 "Use built-in type alias" misfires when using a custom type alias for a CLR type.

### ISSUE THREAD

**Issue by bjornhellander** [DotNetAnalyzers/StyleCopAnalyzers#3424]
The following code triggers SA1121:
```
namespace TestNamespace
{
    using VSCOOKIE = System.UInt32;

    public class TestClass
    {
        public VSCOOKIE Stuff => 0; // SA1121 triggered here
    }
}
```

I suggest that this rule should not trigger on custom type aliases.

Tested using unstable release 1.2.0.304.

**Comment by sharwell** [comment]
Duplicate of #2042
